internal/models/schema-recipe.go

```diff
@@ -18,6 +18,7 @@ import (
 // RecipeSchema is a representation of the Recipe schema (https://schema.org/Recipe).
 type RecipeSchema struct {
 	AtContext       string           `json:"@context"`
+	AtGraph         []interface{}    `json:"@graph,omitempty"`
```

Comment:
I just learned that `@graph` is part of the JSON-LD [spec](https://json-ld.github.io/json-ld.org/spec/latest/json-ld/#dfn-graph-objects). This realization might be huge for scraping because Recipya currently handles `@graph` separately with [parseGraph](https://github.com/reaper47/recipya/blob/main/internal/scraper/scraper.go#L210). 

This being said, no action on your part for this as I will look into it once I come back from vacation on June 17. Only thing is to change the type from `[]interface{}` to `[]any` for consistency and to use identifiers in the [predeclared identifiers block](https://go.dev/ref/spec#Predeclared_identifiers).

Reply:
I got the unmarshalling error from trying to scrape this recipe:
https://thefoodflamingo.com/pistachio-cardamom-tres-leches-cake/

In stepping through with the debugger, it looks like `node.FirstChild.Data` doesn't actually contain a `Recipe` entry at all, so this site might be an edge case.